src/llm/http_payload.hpp

```diff
@@ -27,5 +27,6 @@ struct HttpPayload {
     std::vector<std::pair<std::string, std::string>> headers;
     std::string body;                 // always
     rapidjson::Document* parsedJson;  // pre-parsed body             = null
+    std::string uri;
```

Comment:
it's a minor, but can we place it above headers? 
so we go with metadata first and body after that

Reply:
done